src/Command/Fun/GiphyCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Eve\Command\Fun;
+
+use Eve\Message;
+use Eve\Command\ClientCommand;
+
+use Eve\Command\GiphyClient;
+
+final class GiphyCommand extends ClientCommand
+{
+    /**
+     * @param Message $message
+     *
+     * @return bool
+     */
+    public function canHandle(Message $message): bool
+    {
+        return preg_match('/giphy .+/', $message->text());
+    }
+
+    /**
+     * @param Message $message
+     */
+    public function handle(Message $message)
+    {
+        $client = new GiphyClient();
+
+        $result = $client->getImageFor(
+            str_replace(' ', '+', trim(substr($message->text(), 19))),
+            getenv('GIPHY_TOKEN')
+        );
+
+        if (isset($result)) {

```

Comment:
What are the likely returns from the getImageFor()? Unless it's null, the isset() check isn't too useful, and if it is, then a simple if ($result) would suffice.